tests/app/helpers.py

```diff
@@ -71,40 +71,46 @@ def setup_dummy_suppliers(self, n):
                 )
             db.session.commit()
 
-    def setup_dummy_services_including_unpublished(self, n):
+    def setup_dummy_framework(self, id=123, name='expired', framework='gcloud',
```

Comment:
We probably can get away without this since frameworks are created by migrations and we have the G4 expired one.